DeleteButton.js

```diff
@@ -0,0 +1,19 @@
+export class DeleteButton {
+    constructor(iD, userContainer) {
+        this.url = 'https://jsonplaceholder.typicode.com/users'
+        this.iD = iD + 1;
+        console.log(this.iD)
```

Comment:
get rid of the console.logs. Remeber to NEVER put console.log in the PR-ready code

Reply:
done